mock-mhs-adaptor/src/main/java/uk/nhs/adaptors/mockmhsservice/service/MockMhsService.java

```diff
@@ -84,6 +84,7 @@
     }
 
     private void verifyOutboundMessagePayload(String requestBody) throws JsonProcessingException, MockMHSException {
+        LOGGER.debug("Received outbound MHS request payload:\n{}", requestBody);
```

Comment:
One thing we could point out here is that there's some assumption about the platform, since you're using `\n`. I think we're happy with that, but if not:
```suggestion
        LOGGER.debug("Received outbound MHS request payload:{}{}", System.lineSeparator(), requestBody);
```